governance/polls/Add GUNI-DAI-USDC as a new Vault Type - September 6, 2021.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Add GUNI-DAI-USDC as a new Vault Type - September 6, 2021
+summary: This poll proposes system parameters which could be used to initialize GUNI-DAI-USDC as a new vault type.
+discussion_link: https://forum.makerdao.com/t/g-uni-dai-usdc-uniswap-v3-lp-erc20-collateral-onboarding-application/9026
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Risk Variable
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Add GUNI-DAI-USDC as a new Vault Type - September 6, 2021
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the MakerDAO mandated actors. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, September 6 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding GUNI-DAI-USDC (Gelato) as a new vault type in the Maker Protocol with the parameters below:
+
+### Initial Collateral Parameters
+
+* Underlying Collateral: $collateral_asset
+* [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee): $stability_fee
+* [Liquidation Ratio](https://community-development.makerdao.com/en/learn/governance/param-liquidation-ratio): 105%
+* [Debt Ceiling (`line`)](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling): 10 million DAI
+* [Debt Floor (`dust`)](https://community-development.makerdao.com/en/learn/governance/param-debt-floor): 10,000 DAI
+
+### Liquidation 2.0 Parameters
+
+* [Auction Price Function (`calc`)](https://community-development.makerdao.com/en/learn/governance/param-auction-price-function): Stairstep Exponential?
```

Comment:
Yes. They are all stairstep exponential unless it says otherwise.

Reply:
Good.